src/date-picker/date-picker.scss

```diff
@@ -145,11 +138,37 @@
 }
 
 @mixin iui-time {
+  @include iui-reset;
+  @include iui-font-family;
+  user-select: none;
+  text-align: center;
+  height: $iui-baseline * 26;
```

Comment:
I think I set this height so that the time picker was the same height as the calendar.  If there is no calendar present, I don't suppose we _need_ to set a height.

Reply:
So we show 24 hours without scroll?

Reply:
We would definitely want a scroll, I was more so saying I'm not sure it needs to be capped at this height if no calendar is present.